README.md

```diff
@@ -218,6 +226,124 @@ func userNotificationCenter(
 
 If your user taps on the notification this will be tracked back to Klaviyo as an "Opened Push" event assuming you have implemented the tracking changes discussed above.
 
+## Handling deep linking
+
+There are two use cases for deep linking that can be relevant here - 
+1. When you push a notification to your app with a deep link.
+2. Any other cases where you may want to deep link into your app via SMS, email, web browser etc.
+
+Note that Klaviyo doesn't officially yet support universal links but since there is no validation on the klaviyo front end for URI schemes, one can include universal links but making sure it works as per expectations is on the developer.
+
+In order for deep linking to work, there are a few configurations that are needed and these are no different from what are required for handling deep linking in general and [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on this can be followed in conjunction with the steps highlighted here - 
+
+### Step 1: Register the URL scheme
+
+In order for Apple to route a deep link to your application you need to register a URL scheme in your application's info.plist file. This can be done using the editor that xcode provides in xcode from the Info tab of your project settings (reference screenshot below) or editing the info.plist directly - 
+
+![](example1.png)
+
+The required fields are as following - 
+
+1. **Identifier** - The identifier you supply with your scheme distinguishes your app from others that declare support for the same scheme. To ensure uniqueness, specify a reverse DNS string that incorporates your company’s domain and app name. Although using a reverse DNS string is a best practice, it doesn’t prevent other apps from registering the same scheme and handling the associated links.
+2. **URL schemes** - In the URL Schemes box, specify the prefix you use for your URLs.
+3. **Role** - Since your app will be editing the role select the role as editor
+
+In order to edit the info.plist directly, just fill in your app specific details and paste this in your plist - 
+
+```
```

Comment:
maybe use ```xml here

Reply:
Good call. Done here and there were a few swift code samples that didn't have the ```swift language added. Updated those as well.